tests/unittests/Foundation/ReferenceFoundation/TestNSString.mm

```diff
@@ -595,19 +594,19 @@ guard ensureFiles([@"/tmp/ABC/temp.txt"]) else {
     {
         NSString* path = @"/tmp/";
         auto result = path.stringByDeletingLastPathComponent;
-        ASSERT_OBJCEQ(result, static_cast<NSString*>(_CFGetSlashStr()));
+        ASSERT_OBJCEQ(result, @"/");
```

Comment:
_CFGetSlashStr() returns @"/" on our platform too?

---
In reply to: [72325600](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72325600) [](ancestors = 72325600)